### DISCUSSION TITLE
Does LearningRateMonitor work with deepspeed?

### DISCUSSION THREAD

**Discussion by yuvalkirstain** [Lightning-AI/pytorch-lightning#7777] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I'm trying to run code with PL+Deepspeed. I set my scheduler to be `WarmupDecayLR`, but get a warning `RuntimeWarning: You are using LearningRateMonitor callback with models that have no learning rate schedulers. Please see documentation for configure_optimizers method.`
In the CometML logging I can't see learning rate logs. Is this the expected behaviour?
Thank you! :)

**Comment by kobiso** [comment]
I encountered the same problem and reported in the issue: 
- https://github.com/PyTorchLightning/pytorch-lightning/issues/8520

**Comment by carmocca** [comment, marked as answer]
Thanks for reporting! Closing in favor of the linked issue
